.github/workflows/atd_moped_api.yml

```diff
@@ -17,6 +17,10 @@ on:
       - ".github/workflows/atd_moped_api.yml"
       - ".github/workflows/aws-moped-api-helper.sh"
       - "moped-api/**"
+  workflow_dispatch:
+    inputs:
+      description:
+        default: ""
```

Comment:
I added this to give me the option to trigger the workflow manually, and not only wait for a PR to be opened or a push to main

However I forgot this wouldn't take effect until merging to main, so I ended up having to close a PR and open a new one. Still think its a nice option to have though, so I'm leaving it